app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Api::V1::ReviewsController < ApplicationController
+
+end
```

Comment:
Are we using this endpoint?

Reply:
This endpoint is being used for the reviews form which we're building now. We created a branch off of this one for the feature. So this will be finished in our next PR.